src/cpp/session/modules/SessionHelp.R

```diff
@@ -278,71 +278,72 @@ options(help_type = "html")
    NULL
 })
 
-.rs.addJsonRpcHandler("get_help", function(what, from, type)
+.rs.addFunction("getHelpRpcImpl", function(what, from, type, envir)
```

Comment:
Start of refactoring stuff (moving an inner 'impl' into a separate function; makes debugging more straightforward)